packages/react-next/etc/react-next.api.md

```diff
@@ -5,9 +5,9 @@
 ```ts
 
 import { ComposePreparedOptions } from '@fluentui/react-compose';
-import { IButtonProps } from 'office-ui-fabric-react/lib/Button';
+import { IButtonProps } from '@fluentui/react-next/lib/CompatButton';

```

Comment:
Should this be something more like `'@fluentui/react-next/lib/compat/Button'`

Reply:
I thought we didn't want to allow for imports from internal folders so we restricted that to imports from the overall package, i.e. `@fluentui/react-next` and imports from specific files under `lib`, i.e. `@fluentui/react-next/lib/<component>`.

Reply:
I would prefer `compat/Button`, and we could update our guidance to indicate paths immediately under `compat` are also considered top-level paths that are part of the public API.

Another nice thing about that approach is we could have a file `compat/index` where you could get any compat component exported by its original name. Then only do the renaming when re-exporting from the root (or maybe even better, we don't re-export those things from the root at all).

Reply:
`@fluentui/react-next/lib/compat`

Reply:
That's what I meant as well, I was just being lazy with typing 😆

Reply:
Err to clarify, what I actually meant was something like this, basically mirroring our existing structure that lets you import from either the index (all components) or individual component files:
```
@fluentui/react-next/lib/compat/
  index
  Button
  Checkbox
  ...any other component with compat layer...
```

Reply:
Cool, I went with this!